zone/client.cpp

```diff
@@ -8518,85 +8518,85 @@ void Client::InitInnates()
 	m_pp.InnateSkills[InnateInspect] = InnateEnabled;
 	m_pp.InnateSkills[InnateOpen] = InnateEnabled;
 
-	if (race >= RT_FROGLOK_3) {
```

Comment:
This is a change in behavior

`RT_FROGLOG_3` was `330` and `Race::Froglok` is `26`